BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/SkontoViewModel.swift

```diff
@@ -14,6 +14,12 @@ protocol SkontoViewModelDelegate: AnyObject {
     func didTapDocumentPreview(on viewModel: SkontoViewModel)
 }
 
+extension SkontoViewModelDelegate {
+    func didTapProceed(on viewModel: SkontoViewModel) {
+        // Default implementation
+    }
```

Comment:
[clarification] do we really need this? Until now nobody implemented the current version of the SDK.

Reply:
Its default implementation to avoid implementation for this method in Skonto screen inside RA, because we don't have proceed view on it